eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -0,0 +1,147 @@
+
+  pool:
+    name: $(Pool)
+    # 1es pipeline templates converts `image` to demands: ImageOverride under the hood
+    # which is incompatible with image selection in the default non-1es hosted pools
+    ${{ if eq(parameters.OSName, 'macOS') }}:
+      vmImage: $(OSVmImage)
+    ${{ else }}:
+      image: $(OSVmImage)
+    os: ${{ parameters.OSName }}
+
+  variables:
+    ArmTemplateParameters: '@{}'
+
+  steps:
+  - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+    parameters:
+      paths:
+      - "/*"
+
+  - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
+    parameters:
+      AgentImage: $(OSVmImage)
+
+  - template: /eng/pipelines/templates/steps/use-rust.yml@self
+    parameters:
+      Toolchain: $(RustToolchainName)
+
+  - ${{ parameters.PreSteps }}
+
+  - template: /eng/common/TestResources/build-test-resource-config.yml
+    parameters:
+      SubscriptionConfiguration: ${{ parameters.CloudConfig.SubscriptionConfiguration }}
+      SubscriptionConfigurations: ${{ parameters.CloudConfig.SubscriptionConfigurations }}
+      SubscriptionConfigurationFilePaths: ${{ parameters.CloudConfig.SubscriptionConfigurationFilePaths }}
+      EnvVars:
+        Pool: $(Pool)
+        ${{insert}}: ${{ parameters.EnvVars }}
+
+  - template: /eng/common/TestResources/deploy-test-resources.yml
+    parameters:
+      ${{ if or(parameters.Location, parameters.CloudConfig.Location) }}:
+        Location: ${{ coalesce(parameters.Location, parameters.CloudConfig.Location) }}
+      ServiceDirectory: ${{ parameters.ServiceDirectory }}
+      SubscriptionConfiguration: $(SubscriptionConfiguration)
+      ArmTemplateParameters: $(ArmTemplateParameters)
+      UseFederatedAuth: ${{ parameters.UseFederatedAuth }}
+      ServiceConnection: ${{ parameters.CloudConfig.ServiceConnection }}
+      SubscriptionConfigurationFilePaths: ${{ parameters.CloudConfig.SubscriptionConfigurationFilePaths}}
+      PersistOidcToken: ${{ parameters.PersistOidcToken }}
+      EnvVars:
+        Pool: $(Pool)
+        ${{insert}}: ${{ parameters.EnvVars }}
+
+  # To allow tests to run in parallel to Pack, we don't reuse the PackageInfo directory
+  # from the Pack job. We just let each test leg create its own PackageInfo directory.
+  - template: /eng/common/pipelines/templates/steps/save-package-properties.yml@self
+    parameters:
+      ServiceDirectory: ${{ parameters.ServiceDirectory }}
+      PackageInfoDirectory: $(Build.ArtifactStagingDirectory)/PackageInfo
+
+
+  - ${{ if parameters.UseFederatedAuth }}:
+    - task: AzurePowerShell@5
+      displayName: "Test Packages - Federated Auth"
+      timeoutInMinutes: ${{ parameters.TimeoutInMinutes }}
+      env:
+        CIBW_BUILD_VERBOSITY: 3
+        AZURE_TEST_MODE: 'live'
+        SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+        ${{ if parameters.PersistOidcToken }}:
+          ARM_OIDC_TOKEN: $(ARM_OIDC_TOKEN)
+        ${{ insert }}: ${{ parameters.EnvVars }}
+      inputs:
```

Comment:
If you plan to use AzurePipelineCredential you will also need to set these variables https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/jobs/live.tests.yml#L177

Reply:
We will need those, yes.